code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -1099,6 +1099,18 @@
 	UnregisterSignal(owner, COMSIG_ATOM_UPDATE_OVERLAYS)
 	owner.update_icon()
 
+/datum/status_effect/ineffective_thermoregulation
+	id = "no_temp_reg"
+	alert_type = /atom/movable/screen/alert/status_effect/no_temp_reg
+	duration = 5 SECONDS
+	status_type = STATUS_EFFECT_REPLACE
+
+/atom/movable/screen/alert/status_effect/no_temp_reg
+	name = "Ineffective Thermoregulation"
+	desc = "You can't regulate your temperature! You can't heat up or cool down normally!"
+	icon_state = "stasis"
+//effects handled in _species
```

Comment:
```suggestion
	use_user_hud_icon = TRUE
	overlay_state = "stasis"
```
FYI, I recently refactored screen alerts to allow most of them to use the style of the player interface for the background.